### PR TITLE
kubeadm: add unit test for phases/bootstraptoken/node/token.go

### PR DESCRIPTION
**Pull request by my-git9**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

kubeadm: add unit test for phases/bootstraptoken/node/token.go

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/phases/bootstraptoken/node/token_test.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+		wantErr      bool
+	}{
+		{
+			name:         "token is nil",
+			failIfExists: true,
+			tokens:       []bootstraptokenv1.BootstrapToken{},
+			wantErr:      false,
+		},
+		{
+			name:         "create secret which is not exist",
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/bootstraptoken/node/token_test.go:43]:
```suggestion
			name:         "create secret which does not exist",
```

File: cmd/kubeadm/app/phases/bootstraptoken/node/token_test.go [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+					Token: &bootstraptokenv1.BootstrapTokenString{
+						ID:     "token1",
+						Secret: "token1data",
+					},
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name:         "create multi secret which are not exist",
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/bootstraptoken/node/token_test.go:56]:
```suggestion
			name:         "create multiple secrets which do not exist",
```

File: cmd/kubeadm/app/phases/bootstraptoken/node/token_test.go [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+					Token: &bootstraptokenv1.BootstrapTokenString{
+						ID:     "token3",
+						Secret: "token3data",
+					},
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name:         "create secret which is exist and failIfExists is false",
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/bootstraptoken/node/token_test.go:81]:
```suggestion
			name:         "create secret which exists, failIfExists is false",
```

File: cmd/kubeadm/app/phases/bootstraptoken/node/token_test.go [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+					Token: &bootstraptokenv1.BootstrapTokenString{
+						ID:     "foo",
+						Secret: "bar",
+					},
+				},
+			},
+			wantErr: false,
+		},
+		{
+			name:         "create secret which is exist and failIfExists is true",
```
Comment (neolit123) [inline cmd/kubeadm/app/phases/bootstraptoken/node/token_test.go:94]:
```suggestion
			name:         "create secret which exists, failIfExists is true",
```

**Review by neolit123 — COMMENTED** [review]
thanks for the PR
requesting some minor changes. otherwise, LGTM

/approve
/hold

**Comment by neolit123** [comment]
@my-git9 please update this when possible.

**Comment by my-git9** [comment]
> @my-git9 please update this when possible.

Updated, sorry for the delay

**Pull request merged**